docs/get_started.md

```diff
@@ -18,23 +18,24 @@
    :maxdepth: 1
    :hidden:
    :caption: Additional Configurations
-   
+
    Configurations for GPU <openvino_docs_install_guides_configurations_for_intel_gpu>
    Configurations for NCS2 <openvino_docs_install_guides_configurations_for_ncs2>
    Configurations for VPU <openvino_docs_install_guides_installing_openvino_ivad_vpu>
-   
+   Configurations for GNA <openvino_docs_install_guides_configurations_for_intel_gna>
```

Comment:
@xu-yuan1 do we need to reference to this new page from other places in getting started? Please, advise